platforms/ide/tooling-api/src/main/java/org/gradle/tooling/model/gradle/ResilientGradleBuild.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.tooling.model.gradle;
+
+import org.gradle.api.Incubating;
+import org.gradle.tooling.model.BuildIdentifier;
+import org.gradle.tooling.model.BuildModel;
+import org.gradle.tooling.model.DomainObjectSet;
+import org.gradle.tooling.model.Model;
+import org.jspecify.annotations.NullMarked;
+
+/**
+ * Provides information about the structure of a Gradle build.
+ * This is a "resilient" version of {@link GradleBuild} that provides information about build failures.
+ *
+ * @since 9.2.0
+ */
+@Incubating
+@NullMarked
+public interface ResilientGradleBuild  extends Model, BuildModel {
+
+    /**
+     * Returns the identifier for this Gradle build.
+     *
+     * @since 9.2.0
+     */
+    @Override
+    BuildIdentifier getBuildIdentifier();
```

Comment:
❓ What happened to the idea of making `ResilientGradleBuild` aggregate a `GradleBuild` instead of replicating its interface?

Reply:
I think we said, that this will be a separate effort?